install-dev/theme/js/process.js

```diff
@@ -94,7 +94,7 @@ function process_install(step)
 			// An error occured during this step
 			else
 			{
-				install_error(step, (json) ? json.message : '');
+				install_error(step, (json) ? json.message : json);
```

Comment:
Woups, this is a change I wanted to remove.
If we do not have json returned, we won't go in the success function and a `parserror` will be caught in the next lines.